oic.r.csr.raml

```diff
@@ -0,0 +1,35 @@
+#%RAML 0.8
+title: OICSecurityCsrResource
+version: v1.0-20150819
+schemas:
+  - Csr: !include schemas/oic.r.csr.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+            enum: ["oic.if.baseline"]
+
+/oic/sec/csr:
+  description: |
+    This resource specifies a Certificate Signing Request.
+
+  displayName: Certificate Signing Request
+  is: [ interface ]
+
+  get:
+    description: |
+      Retrieves CSR data.
+    responses:
+      200:
+        body:
+          application/json:
+            schema: Csr
+            example: |
```

Comment:
Atom is acting up on me and refuses to validate any of the examples. But other than the missing comma, I hope this one is OK. Everything else checks out...